src/util.h

```diff
@@ -85,10 +85,16 @@ extern bool v8_initialized;
 // whether V8 is initialized.
 void LowMemoryNotification();
 
-// The slightly odd function signature for Assert() is to ease
-// instruction cache pressure in calls from CHECK.
+// The reason that Assert() takes a struct argument instead of individual
+// const char*s is to ease instruction cache pressure in calls from CHECK.
+struct AssertionInfo {
+  const char* filename;
+  unsigned int linenum;
```

Comment:
this should be `unsigned long int` to avoid unnecessary padding on x64. i.e.
```asm
main::b:
        .quad   .LC1
        .long   20
        .zero   4
        .quad   .LC2
        .quad   .LC3
```
https://godbolt.org/z/l-PU0z

Reply:
@refack I’m not sure I understand – the data that ends up in the binary is exactly the same for x64 (assuming line numbers don’t exceed 2^32), and on 32-bit platforms a simple 32-bit integer is also ideal.